deploy/role.yaml

```diff
@@ -9,10 +9,7 @@ rules:
   resources:
     - storageclasses
   verbs:
-    - get
-    - list
-    - watch
-    - create
+    - '*'
```

Comment:
Please let's try not to use blanket all permissions. If you want to have permissions for update, just add update to the list. I know there are other similar permissions, which we should be revisiting and correcting for just the permissions we need. But let's try not to make this more work for ourselves later.

Reply:
Sure.